agent/ebs/watcher_linux.go

```diff
@@ -0,0 +1,22 @@
+//go:build linux
+// +build linux
+
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//      http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package ebs
+
+const (
+	csiDriverSocketAddress = "/var/run/ecs/ebs-csi-driver/csi-driver.sock"
```

Comment:
Is this being used anywhere? I know we use `hostMountDir` that's already being used in the existing implementation and we're declaring the value of the variable differently between linux and windows but I don't recall if we use `csiDriverSocketAddress`.

Reply:
You are correct. I will remove this file.